source/agora/node/admin/AdminInterface.d

```diff
@@ -193,8 +193,10 @@ public class AdminInterface : NodeControlAPI
         EncryptionKey encryptionKey = EncryptionKey(
             app,
             Height(height),
-            key
+            key,
+            this.key_pair.address,
```

Comment:
Note that you are still hashing `Hash.init` (see L188)

Reply:
https://github.com/bosagora/agora/pull/2113
It is being applied by other PR.
If it is merged, I will rebase the PR.